src/sql/parts/notebook/models/sparkMagicContexts.ts

```diff
@@ -0,0 +1,194 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
```

Comment:
There's a decent bit of rewriting done in this file, will need to test / investigate this as we enable more code